aea/manager/manager.py

```diff
@@ -261,13 +267,16 @@ def _run_agent(
         t: Optional[Thread] = None
         r: Optional[Exception] = None
         run_stop_thread: bool = True
+        # set a new event loop, cause it's a new process
+        asyncio.set_event_loop(asyncio.new_event_loop())
         try:
             aea = agent_alias.get_aea_instance()
 
             def stop_event_thread() -> None:
                 try:
                     while run_stop_thread:
-                        if stop_event.wait(0.05) is True:
+                        if stop_event.wait(0.01) is True:
+                            print("Stop event set!")
```

Comment:
why not logging module?

Reply:
it's a subprocess, not sure about logging. anyway it was added for debug purposes and i can remove it now